packages/ag-charts-enterprise/src/series/histogram/histogramSeries.ts

```diff
@@ -567,32 +567,25 @@ export class HistogramSeries extends CartesianSeries<_Scene.Rect, HistogramNodeD
     }
 
     override animateEmptyUpdateReady({ datumSelections, labelSelections }: HistogramAnimationData) {
-        const fns = prepareBarAnimationFunctions(collapsedStartingBarPosition(ChartAxisDirection.Y, this.axes));
+        const fns = prepareBarAnimationFunctions(collapsedStartingBarPosition(true, this.axes));
```

Comment:
I'd prefer to keep this parameter using `ChartAxisDirection` here, as it's better typed than `boolean` and harder to mis-match in the middle of a refactor?.

Reply:
Discussed offline.